include/ck_tile/ops/fmha/pipeline/block_fmha_pipeline_qs_ks_vs.hpp

```diff
@@ -341,6 +335,10 @@ struct BlockFmhaPipelineQSKSVS
                 gemm_0(s_acc, q_lds_window, k_lds_window);
             }
 
+            __builtin_amdgcn_sched_barrier(0);
+
+            const auto v_prefetch = load_tile(v_dram_window); // prefetch load v tile
+
```

Comment:
should we add another `__builtin_amdgcn_sched_barrier(0)` here to make sure V prefetcch is schedued as early as possible?

Reply:
Not tested,  just direct added this.  Let me have a try to remove this.

Reply:
Remove `__builtin_amdgcn_sched_barrier(0)`, it does not affect the testing result